src/script/ceph-backport.sh

```diff
@@ -1074,6 +1074,7 @@ function try_known_milestones {
         quincy) mn="15" ;;
         reef) mn="16" ;;
         squid) mn="17" ;;
+        tentacle) mn="18" ;;
```

Comment:
are you sure `18` is correct? i tried the `curl --verbose -X GET https://api.github.com/repos/ceph/ceph/milestones` command mentioned above and milestone 18 corresponds to "v17.2.8" there. tentacle seems to be 31:
```
$ curl --silent https://api.github.com/repos/ceph/ceph/milestones/18 | jq .title
"v17.2.8"
$ curl --silent https://api.github.com/repos/ceph/ceph/milestones/31 | jq .title
"tentacle"
```